match/lib/match/nuke.rb

```diff
@@ -173,6 +178,77 @@ def prepare_list
       self.files = certs + keys + profiles
     end
 
+    def filter_by_cert
+      # Force will continue to revoke and delete all certificates and profiles
+      return if self.params[:force]
+      return if self.certs.count < 2
+
+      # Print table showing certificates that can be revoked
+      puts("")
+      rows = self.certs.each_with_index.collect do |cert, i|
+        cert_expiration = cert.expiration_date.nil? ? "Unknown" : Time.parse(cert.expiration_date).strftime("%Y-%m-%d")
+        [i + 1, cert.name, cert.id, cert.class.to_s.split("::").last, cert_expiration]
+      end
+      puts(Terminal::Table.new({
+        title: "Certificates that can be revoked".green,
+        headings: ["Option", "Name", "ID", "Type", "Expires"],
+        rows: FastlaneCore::PrintTable.transform_output(rows)
+      }))
+      puts("")
+
+      if UI.confirm("Do you want to nuke specific certificates and their associated profiles?")
```

Comment:
Does it make sense to add an explanation that `Declining will nuke all the listed certificates and their associated profiles`

Reply:
Hi @joshdholtz, sure, I'll send you an email.